### ISSUE TITLE
Sentry wizard overrides nvm configuration

### ISSUE THREAD

**Issue by troZee** [getsentry/sentry-wizard#518]
### Environment

MacOS

### Steps to Reproduce

1. Install sentry wizard 
2. run `nvm ls`
3. run `nvm use 18.12.0 `
4. run `nvm current`

### Expected Result

sentry-wizard should not install node globally, which overrides nvm settings 

### Actual Result

If the node is installed, we should use the currently installed node. 

```
➜  ~ brew uninstall node
Error: Refusing to uninstall /opt/homebrew/Cellar/node/21.3.0
because it is required by sentry-wizard, which is currently installed.
You can override this and force removal with:
  brew uninstall --ignore-dependencies node
```

**Comment by Lms24** [comment]
@romtsn I believe this is related to the brew target. Would you mind taking a look at this issue?

**Comment by romtsn** [comment]
@Lms24 Sure, looks like we have to write a custom [requirement](https://rubydoc.brew.sh/Requirement) that checks for an existing Node version and uses it, or otherwise installs the one from brew. The min supported Node version by the wizard is at 14.18 now, right?

**Comment by Lms24** [comment]
> The min supported Node version by the wizard is at 14.18 now, right?

@romtsn yup. We might bump it in the future given Node 14 is EOL since a while but right now, it's `>=1418.0`.

**Comment by kahest** [comment]
@romtsn @Lms24 is this still relevant?

**Comment by romtsn** [comment]
yes most likely it's still an issue

**Comment by romtsn** [comment]
We could also change the formula to pull the new self-contained binary instead which would not carry the node dependency at all.
